driver-core/src/main/com/mongodb/operation/CreateCollectionOperation.java

```diff
@@ -251,6 +335,11 @@ private BsonDocument asDocument() {
         if (indexOptionDefaults != null) {
             document.put("indexOptionDefaults", indexOptionDefaults);
         }
+        if (validator != null) {
+            document.put("validator", validator);
+            document.put("validationLevel", new BsonString(validationLevel.getValue()));
```

Comment:
Along with earlier comments: allow for unspecified level and action.  I checked, and the server doesn't require them.